docs/cli.md

```diff
@@ -33,137 +33,181 @@ CLI commands can be accessed using the Docker/Singularity image.
     If you have instead locally built an image, replace `neurobagel/bagelcli` in commands with your built image tag.
 
 ### Input files
-To run the CLI on a dataset you have annotated, you will need:
+The Neurobagel CLI can compile information from several different data sources to create a single harmonized representation of subject data. To run the CLI on a dataset, you will need:
 
 <div class="annotate" markdown>
 - [A phenotypic TSV](./data_prep.md)
-- [A corresponding phenotypic JSON data dictionary](./dictionaries.md)
+- [A Neurobagel JSON data dictionary](./dictionaries.md) for the TSV
 - (Optional) The imaging dataset in [BIDS](https://bids-specification.readthedocs.io/en/stable/) format, if subjects have imaging data available (1)
+- (Optional) A TSV containing subject statuses for any image processing pipelines that have been run, following the [Nipoppy processing status file schema](https://nipoppy.readthedocs.io/en/latest/schemas/index.html#bagel-file) (2)
+    
+Note: The CLI does not require a BIDS dataset to accept a processing status file, and vice versa.
```

Comment:
I find this note confusing, but I also don't have a short way that is clearer. How about we just scrap the note, as the `(Optional)` part is already saying the same thing